### PR TITLE
React & Redux group project: Space Travelers' Hub

### PR DESCRIPTION
**Pull request by a-laarabi**
# What?
In This project I @a-laarabi and my partner @Lusindiso has been working the whole week to build a web application for a company that provides commercial and scientific space travel services. This application will allow users to book rockets and join selected space missions.

## Highlights 🔥 

- Use React documentation.
- Use React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write unit tests with React Testing Library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions, and reducers in React.
- Perform a code review for a team member.

- Ps : 
   - We reviewed all the pull requests that we created.
   - We created two testing files to test our components.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,69 @@
-# spaceTravelers-Hub
\ No newline at end of file
+# Space Travelers' Hub
+
+> ![space travellers](https://user-images.githubusercontent.com/90258833/176904082-2b439a27-4913-489c-95ea-5dfbf5b98f08.png)
+
+> Space Travelers' Hub is to a web application for a company that provides commercial and scientific space travel services. This application allows users to book rockets and join selected space missions.
+
+## Features
+
```
Comment (tufoinnkuo10) [inline README.md:8]: - [optional]. you can deploy your app on Netlify to ease the live preview

File: README.md [hunk 2]
```diff
@@ -1 +1,69 @@
...
+👤 **Lusindiso Ntanjana**
+
+- GitHub: [@Lusindiso](https://github.com/Lusindiso)
+- Twitter: [@LusindisoNt](https://twitter.com/LusindisoNt)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/lusindisontanjana/)
+
+👤 **Anasse LAARABI**
+
+- GitHub: [@a-laarabi](https://github.com/a-laarabi)
```
Comment (tufoinnkuo10) [inline README.md:53]: - [ optional] it would be nice to complete the second authors' info

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Team

Great job so far. There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ design and profile looks ok
✅Rocket and mission page ok

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by medaminedev66 — APPROVED** [review]
Hi @a-laarabi and @Lusindiso 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 💯 
Congratulations! 🎉 

### Highlights
- The project is deployed properly.
- Your web app works as expected and runs without errors.
- Your design looks good.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**